packages/gestalt/src/__snapshots__/TapArea.test.js.snap

```diff
@@ -163,7 +163,7 @@ exports[`TapArea supports press style 1`] = `
   onTouchMove={[Function]}
   onTouchStart={[Function]}
   role="button"
-  tabIndex="0"
+  tabIndex={0}
```

Comment:
@AlbertCarreras heads up that the `tabIndex` change will result in a lot of snapshot updates in the Pinterest codebase